packages/core/src/actions/bundler/estimateUserOperationGas.ts

```diff
@@ -1,16 +1,18 @@
 import type { Address, Chain, Client, StateOverride, Transport } from "viem";
 import type { BundlerRpcSchema } from "../../client/decorators/bundlerClient";
+import type { EntryPointVersion } from "../../entrypoint/types";
 import type {
   UserOperationEstimateGasResponse,
   UserOperationRequest,
 } from "../../types";
 
 export const estimateUserOperationGas = async <
+  TEntryPointVersion extends EntryPointVersion,
```

Comment:
nit: for consistency with the alchemy simulate UO method and the other methods, I think this should go second in the list. Also this is for the same reason as my previous comments on the original PR -- if anyone was explicitly typing this method for w/e reason, this is a breaking change for them